lib/twilio/audiohelper.ts

```diff
@@ -342,6 +348,35 @@ class AudioHelper extends EventEmitter {
     });
   }
 
+  /**
+   * Adds an {@link AudioProcessor}. If an {@link AudioProcessor} exists in the
```

Comment:
In my opinion, the wording here seems to imply "If an AudioProcess (already) exists in the..." due to grammatical structure. My gut instinct reading this is that the functionality only works on the second or further invocation of the function, which I am assuming is not true. I'm trying to think of how I would word this... maybe something like

```
Adds an AudioProcessor.

Once the AudioProcessor contains at least one AudioProcessor, the AudioHelper routes the input audio stream to the AudioProcessor.
```

Reply:
I understand that currently we do not support multiple AudioProcessors, so maybe you'd want to reword that.

Reply:
I'm not sure what you mean by "the functionality only works on the second or further invocation of the function". This was reviewed with the whole team. And another one with you in the IDL. I'm just wondering why the confusion now? Can you elaborate?